src/ansys/tools/versioning/utils.py

```diff
@@ -276,3 +283,128 @@ def wrapper(self, *args, **kwargs):
         return wrapper
 
     return decorator
+
+
+class Version(int):
```

Comment:
Thinking about the usage of this class, I would suggest renaming it or ``SemanticVersion``, as this is the convention we follow in PyAnsys.

In addition, a ``SemanticVersion`` should include the major, minor, patch and suffix numbers. The more explicit we are when creating instances of this class, the better.

Reply:
Version is only a digit/number ... not the full X.X.X

But I think I'm going to have to implement it. Let's say:

SemanticVersion -> "x.x.x"
NumberVersion -> "x"

Are we ok with that?

Reply:
I agree with the implementation of the SemanticVersioning class. If we still need VersionNumber then go ahead with it :)

Reply:
Done. Check the last push.

Reply:
wait a minute... 

> SemanticVersion should include the major, minor, patch and suffix numbers.

What is suffix??